### PR TITLE
Added instructions on how to upload Gallery application to play store.

### PR DESCRIPTION
**Pull request by JoseAlba**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,3 +129,6 @@ flutter pub run grinder update-code-segments
     * Update the description to include what changes have been done since the
     last release.
     * Publish the release.
+
```
Comment (guidezpl) [inline README.md:132]: Can you replace the instructions in step 4?

Comment (JoseAlba) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -124,6 +124,7 @@ flutter pub run grinder update-code-segments
     * Create the app bundle with `flutter build appbundle`.
     * Upload to the Play store console.
     * Publish the Play store release.
+    * Upload to Google play store [instructions](https://docs.google.com/document/d/1M3NUVdVo0jHXkyTzZ8d-YRDA6gCqOl0JWV8enbwJF2w/edit?usp=sharing).
```
Comment (rami-a) [inline README.md:127]: I think these steps are covered in the google doc, so I think step 4 can just be shortened to include a link to the doc  with no steps.

**Review by rami-a — APPROVED** [review]
lgtm

**Review by guidezpl — APPROVED** [review]

**Comment by guidezpl** [comment]
Deprecating this with a PR to automate the whole thing :)

**Pull request closed without merging**